requirements.txt

```diff
@@ -1,11 +1,3 @@
-httmock
-wooper
-PyRTF3
-reportlab==3.5.53
-xhtml2pdf
-werkzeug>=0.9.4,<=0.11.15
-email-validator==1.0.5
-urllib3<1.26
 -e .
-git+git://github.com/superdesk/superdesk-planning.git@1.10#egg=superdesk-planning
+superdesk-planning==1.33.2
```

Comment:
It might be better to have these two dependencies in `setup.py:install_requires`, especially if we publish this to PyPi as a package (example usage would be for a customer repo)

Reply:
done